wiki/Gameplay/Multiplayer/en.md

```diff
@@ -1,434 +1,380 @@
-The player should enter the correct password to access the locked Match Setup.
-Alternatively, the player can request an invite from the Match Setup's host directly to bypass the locked password requirement.
+To enter a match, click on one of the desired rows.
 
----
+### General options
 
-The three buttons above the Chat Console are special commands a player can use while in the Lobby, reading from left-to-right:-
+The three buttons above the [chat console](/wiki/Client/Interface/Chat_console) can be used in the main lobby:
 
 | Title | Description |
 | :-: | :-- |
-| `Back to Menu` | Self-explanatory; **leave the lobby and return to the main menu**. |
-| `New Game` | **Preparation to create a new Match Setup** |
-| `Quick Join` | Place the player to an **available Match Setup** *randomly* based on **current pp rank range**. |
-
-The [Chat Console](/wiki/Client/Interface/Chat_console) will be available at the bottom of the screen and connected to `#lobby` by default.
+| `Back to Menu` | Leave the lobby and return to the main menu. |
+| `New Game` | Create a new game. See below for more details. |
+| `Quick Join` | Join an available match *randomly* based on the player's current pp rank range. |
 
-## New game
+## Creating a new game
 
-![](img/Multi_new_game.jpg "Setting up configuration for a New Game")
+![](img/Multi_new_game.jpg "Setting up configuration for a new game")
 
 | Title | Description |
 | :-: | :-- |
-| **Game Name** | **Name of the Match Setup.** Default to `{accountname}`'s game. To make the Match Setup private, enable the `Require Password to join` button. |
-| **Password** | **Password to enter the Match Setup.** Only appears if the `Require Password to join` button was enabled. Match Setup game mode symbol will be shown with a lock. |
-| **Max players** | **Total number of players allowed to join the Match Setup.** Default to 8 players, with range from 2 to a total of 16 players, including the player as host. It can be adjusted later by locking/unlocking the slots. |
+| `Game Name` | The name of the match. The default value is `{account name}'s game`. |
+| `Require password to join` | Make the match private. |
+| `Password` | Set a password for the match. Only appears if the `Require password to join` button was enabled. |
+| `Max Players` | Total number of players allowed to join the match. The default is 8 players, with a range from 2 to a total of 16 players, including the player as host. It can be adjusted later by locking/unlocking the slots. |
 
-Press the `1. Start Game` button to **create the Match Setup** with the **player as the host** of the Match Setup and the **beatmap playing in the jukebox as the match's beatmap/difficulty**.
+Pressing the `1. Start Game` button will create the match with the beatmap playing in the jukebox as the match's beatmap. The button `2. Cancel` brings the player back to the lobby.
 
-## Match Setup
+## Match setup
```

Comment:
this section mixes neutral and informal writing styles with the `here's what you can see` additions